### DISCUSSION TITLE
TypeError: britecharts.groupedBar is not a function

### DISCUSSION THREAD

**Discussion by Lennen** [britecharts/britecharts#1020] [General]
TypeError: britecharts.groupedBar is not a function. My code worked earlier, but now there is a such error, I can't use groupedBar. What happened, can I see some example to easily rewrite my old code?

**Comment by Golodhros** [comment]
Hi @Lennen, thanks for mentioning this!
Which version of Britecharts are you using?

**Comment by Lennen** [comment]
<script ***@***.***/dist/umd/donut.min.js" type="text/javascript"></script>
etc... But with donut all is ok, but not with the Bar, GroupedBar and StackedBar, these 3 types do not working
    On Tuesday, 14 March 2023 at 20:15:31 GMT+3, Marcos Iglesias ***@***.***> wrote:  
 
 


Hi @Lennen, thanks for mentioning this!
Which version of Britecharts are you using?

**Reply by Golodhros** [reply]
Oh, I need that numbers around the "@"!
